pkg/controller/util/owner.go

```diff
@@ -0,0 +1,21 @@
+package util
+
+import (
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetStorageClusterInitializationOwnerRef returns the owner references instance of the storageClusterInitialization
+func GetStorageClusterInitializationOwnerRef(object *ocsv1.StorageClusterInitialization) metav1.OwnerReference {
```

Comment:
Now that you're using the controllerutil function, this is not needed anymore. So please remove this.

Reply:
done